From aad367a4661b7b15c4d8a69de6d314ea173efd52 Mon Sep 17 00:00:00 2001
From: David King <davidk@gnome.org>
Date: Mon, 30 Jun 2014 11:31:08 +0100
Subject: [PATCH] Match desktop file name in AppData

Update the AppData reference to the desktop file, as it was not changed
when the desktop file was renamed.
---
 data/org.gnome.Logs.appdata.xml.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/org.gnome.Logs.appdata.xml.in b/data/org.gnome.Logs.appdata.xml.in
index f085894..2922115 100644
--- a/data/org.gnome.Logs.appdata.xml.in
+++ b/data/org.gnome.Logs.appdata.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2013 David King <davidk@gnome.org> -->
+<!-- Copyright 2013, 2014 David King <davidk@gnome.org> -->
 <application>
-    <id type="desktop">gnome-logs.desktop</id>
+    <id type="desktop">org.gnome.Logs.desktop</id>
     <licence>CC0</licence>
 <!-- Both taken from desktop file.
     <name>Logs</name>
-- 
2.0.0

